app/Helpers/database/forum.php

```diff
@@ -230,9 +230,15 @@ function notifyUsersAboutForumActivity(int $topicID, string $topicTitle, string
         "
     );
 
+    $payload = null;
+    $comment = ForumTopicComment::find($commentID);
+    if ($comment) {
+        $payload = nl2br(Shortcode::stripAndClamp($comment->Payload, previewLength: 1000, preserveWhitespace: true));
```

Comment:
`Shortcode::render` mostly works here. It does the formatting and creates clickable links, but there's a lot of extra decorations for the tooltips that can't function in the email, so I opted to unformat everything.